dotcom-rendering/src/web/components/TrendingTopicComponent.tsx

```diff
@@ -0,0 +1,68 @@
+import { css } from '@emotion/react';
```

Comment:
I think we don't often suffix components with Component at the end of the filename (I believe this is mostly used where the components are a capi/fe element), so we could probably just name this `TrendingTopics.tsx`